packages/backend-tasks/src/tasks/PluginTaskSchedulerImpl.ts

```diff
@@ -24,6 +24,8 @@ import {
   TaskScheduleDefinition,
 } from './types';
 import { validateId } from './util';
+import { DB_TASKS_TABLE, DbTasksRow } from '../database/tables';
```

Comment:
I realize - now this package needs a changeset too

Reply:
oh yeah! added!